apps/deploy-web/tests/ui/fixture/testing-helpers.ts

```diff
@@ -0,0 +1,41 @@
+import type { Locator, Page } from "@playwright/test";
+
+export const getWalletSelectorDropdown = async (page: Page) => {
+  const buttons = await page.$$("#popup-layout button");
+  return buttons.length ? buttons[0] : null;
+};
+
+export const clickWalletSelectorDropdown = async (page: Page) => {
+  return (await getWalletSelectorDropdown(page))?.click();
+};
+
+export const clickCreateNewWalletButton = async (page: Page) => {
+  const createNewWalletButton = page.getByText(/create new wallet/i);
+  await createNewWalletButton.waitFor({ state: "visible", timeout: 10_000 });
+  await createNewWalletButton.click();
+};
+
+export const fillWalletName = async (page: Page, name: string) => {
+  const input = await waitForSelector(page, 'input[placeholder="Enter wallet Name"]');
+  return await input.fill(name);
+};
+
+export const clickCreateWalletButton = async (page: Page) => {
+  const button = await waitForSelector(page, 'button:has-text("Create Wallet")');
```

Comment:
```suggestion
  const button = await waitForLocator(page.getByRole('button', { name: /Create Wallet/ });
```

https://playwright.dev/docs/locators#locate-by-role